backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -988,6 +1001,18 @@ bool SurfaceSdlGraphicsManager::loadGFXMode() {
 	handleResize(_videoMode.hardwareWidth, _videoMode.hardwareHeight);
 #endif
 
+#ifdef MIYOOMINI
+	_realHwScreen = _hwScreen;
```

Comment:
I wonder if some of these Miyoo (and RS90) ifdefs can go into a subclass, similarly to what the GPH, Maemo and OpenPandora backends do.

Reply:
Done